### ISSUE TITLE
Hidden fields should not be added if existing already

### ISSUE THREAD

**Issue by larsblumberg** [jackocnr/intl-tel-input#1843]
### Plugin version
v24.6.0

### Steps to reproduce
1.  set hiddenInput: () => ({ phone: "phone_full", country: "phone_country" })`
2.  provide these hidden inputs yourself in the form (there are advantages for certain scripting like referencing these before the library creates them dynamically)

### Expected behaviour
- library does not add hidden inputs a 2nd time, because we already set them up

### Actual behaviour
- `value` of above inputs gets correctly set
- but additional hidden inputs are being added

![image](https://github.com/user-attachments/assets/d88c0544-f32b-45fe-ba1a-ba43f8557723)

### Initialisation options

```
  intlTelInput(input, {
    hiddenInput: () => ({ phone: "phone_full", country: "phone_country" }),
    loadUtilsOnInit: () => import("intl-tel-input/utils"),
  });
```

**Comment by jackocnr** [comment]
I'm onboard with this idea. We just need to check if the hidden inputs already exist (within the form) before creating them. 

I don't have time to implement this myself right now, but I would welcome a pull request.

**Comment by jackocnr** [comment]
Released in v25.0.1
